packages/math-input/src/components/__tests__/integration.test.tsx

```diff
@@ -91,7 +97,7 @@ describe("math input integration", () => {
         ).toBeInTheDocument();
     });
 
-    it("doesn't show the keypad initially", () => {
+    it("doesn't show the keypad initially", async () => {
```

Comment:
I used some of the automation John figured out for migrating webapp and so there might be a few tests in this PR that are migrated to be async that don't actually call any async functions. In practice, this didn't seem to bother. I'm happy to roll back these if the reviewer has strong feelings. :)